ui/preference/list_preference.go

```diff
@@ -103,17 +121,22 @@ func (lp *ListPreference) Layout(gtx layout.Context, body layout.Dimensions) lay
 			return body
 		}),
 		layout.Stacked(func(gtx layout.Context) layout.Dimensions {
-			return lp.modal(gtx)
+			return lp.printModal(gtx)
 		}),
 	)
 }
 
-func (lp *ListPreference) modal(gtx layout.Context) layout.Dimensions {
+func (lp *ListPreference) printModal(gtx layout.Context) layout.Dimensions {
 	w := []layout.Widget{
 		func(gtx layout.Context) layout.Dimensions {
 			txt := lp.theme.H6(values.String(lp.titleStrKey))
 			txt.Color = lp.theme.Color.Text
-			return txt.Layout(gtx)
+			return layout.Flex{Axis: layout.Horizontal, Spacing: layout.SpaceBetween}.
```

Comment:
for lines 131, 132 and others using `lp.theme`, you will need to change those to `lp.Theme`